x-pack/plugins/security_solution/public/resolver/models/indexed_process_tree/__snapshots__/isometric_taxi_layout.test.ts.snap

```diff
@@ -212,6 +212,10 @@ Object {
     },
     Object {
       "metadata": Object {
+        "elapsedTime": Object {
```

Comment:
These changed because the `timestampSafeVersion` now retrieves the `@timestamp` field on the legacy event type as well. Previously, we were the isometric code was receiving an empty string which resulted in `null` be used for this field so it was excluded from the metadata.